broker/src/main/java/io/zeebe/broker/system/configuration/BackpressureCfg.java

```diff
@@ -12,6 +12,11 @@
   private boolean enabled = true;
   private boolean useWindowed = true;
   private String algorithm = "vegas";
+  private final AIMDCfg aimd = new AIMDCfg();
```

Comment:
Optional: I know you only changed the class, bit there is one thing bothering me about the code that is already here. Feel free to ignore or implement this comment.

the algorithm is stored as a String internally. The  string is not checked in the setter. It will throw exceptions in the getter if the wrong string is passed in. In my mind there should be at least be a check whether the string can be parsed to an enum. And once we do that, we might as well use the enum as internal representation